docs/hoon/twig/cen-call/sig-open.md

```diff
@@ -13,6 +13,10 @@ sort: 3
 :rap(p :make(q +6 r))
 ```
 
+```
+=<(p %=(q +6 r))
+```
+
```

Comment:
Ah yes, that was on the list. Well, it's =+ %= (or =+ %_ maybe?), with
limb-wrapping to make the expansion hygienic.

On Thursday, 23 June 2016, Joe Bryan <notifications@github.com> wrote:

> In docs/hoon/twig/cen-call/sig-open.md
> <https://github.com/urbit/docs/pull/55#discussion_r68298123>:
>
> > @@ -13,6 +13,10 @@ sort: 3
> >  :rap(p :make(q +6 r))
> >  ```
> >
> > +```
> > +=<(p %=(q +6 r))
> > +```
> > +
>
> Thanks, that's helpful context. And, ironically enough, %* is, in fact,
> undocumented ;)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/urbit/docs/pull/55/files/fb89b530694cb031b6e33d0b360f304e6cb77089#r68298123>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABxXhmY86swk53X8gP3A2EbbUmSM74XYks5qOtyWgaJpZM4I9D6c>
> .
>